cli/test/test_runner_host.py

```diff
@@ -1,13 +1,19 @@
 import os
+import time
 import unittest
 
+from subprocess import Popen
+
+import psutil
```

Comment:
`psutil` is not a pure-python library (it gets compiled when installed), so this makes popper less portable and harder to install. Since this is only a test dependency, we could add it as a test dependency like we had before (where we would install using `pip install cli/[dev]`)